src/main/java/aj/Storage.java

```diff
@@ -0,0 +1,149 @@
+package aj;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Storage class responsible for CRUD operations with database.
+ */
+public class Storage {
+
+    Parser parser;
+    final String FILEPATH;
+
+    /**
+     * Reads data from database, creates the necessary Task objects and returns an Arraylist of Tasks.
+     *
+     * @return Arraylist of Tasks.
+     */
+    public List<Task> initialiseData() { // get data from file, create necessary task objects and returns an array of
+        // task
+//        String localDir = System.getProperty("user.dir");
+//        File file = new File(localDir + "/src/main/data/actualData.txt");
```

Comment:
Perhaps you could remove commented out code so that it will look cleaner

Reply:
I agree with ryantzr1's comment on this. Perhaps you could do that for other parts of the different files as well.